### DISCUSSION TITLE
No puedo cagar un kernel

### DISCUSSION THREAD

**Discussion by DanArturo** [LaunchX-InnovaccionVirtual/CursoIntroPython#128] [General]
Hola alguien me puede ayudar?
No puedo cargar el kernel y no me deja ejecutar código.
De un día para otro paso esto, anterior mente funcionaba bien.
![Captura de pantalla 2022-02-11 031600](https://user-images.githubusercontent.com/99112677/153565955-59a25ef5-688b-4035-a04f-b6af9a5b7373.png)

**Comment by BarajasAngel** [comment]
Tienes instalado Python?

**Reply by DanArturo** [reply]
si. ayer funcionaba correctamente y hoy que prendí ni compu y n sirvió

**Reply by BarajasAngel** [reply]
Te muestra algún error por consola ?

**Reply by DanArturo** [reply]
> Te muestra algún error por consola ?

no

**Reply by BarajasAngel** [reply]
Intenta ejecutar el Notebook con F5 para ver si muestra algún error

**Reply by DanArturo** [reply]
> Intenta ejecutar el Notebook con F5 para ver si muestra algún error

reinicie la pc y se arreglo, pero  gracias por ayudarme :)

**Comment by YessyRuiz** [comment]
Buenos dias , eso te puede ocurrir porque esta seleccionado como texto plano en vez de python donde estas escribiendo el codigo

![ejemplo 1](https://user-images.githubusercontent.com/99302667/153586853-36e88af1-84d9-45d5-aad6-2ec6c7de05f2.png)

Se va abrir un panel con los lenguajes que soporta jupyter

![ejemplo 2](https://user-images.githubusercontent.com/99302667/153587393-8abad744-6455-4f9b-bfdc-0a36d2424b10.png)

ya con eso no deberia dar problema al momento de correr el codigo.
